pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxSendOperator.java

```diff
@@ -115,6 +125,11 @@ public MailboxSendOperator(MailboxService<TransferableBlock> mailboxService,
     _exchange = blockExchangeFactory.build(mailboxService, receivingMailboxes, exchangeType, keySelector, splitter,
         deadlineMs);
 
+    _collationKeys = collationKeys;
+    _collationDirections = collationDirections;
+    _sortOnSender = sortOnSender;
+    Preconditions.checkState(!sortOnSender, "Sort on send is not yet implemented!");
```

Comment:
do not check this on the sender operator. only check in the sending node side. the support precondition is a planner constrain not a runtime constrain

Reply:
done